src/app/screens/LogIn/layout.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+import './styles.css';
+
+// Validations
+const required = value => (value ? undefined : 'Required Field');
+const passwordLength = value => (value && value.length < 8 ? 'Must be at least 8 chars' : undefined);
+const emailRegex = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i;
+const email = value => (value && !emailRegex.test(value) ? 'Invalid email address' : undefined);
+
+// eslint-disable-next-line
+const renderField = ({ input, label, type, meta: { touched, error } }) => (
```

Comment:
move this component into `components` folder like a `TextInput` component

Reply:
Done